slackbot/dispatcher.py

```diff
@@ -17,10 +17,11 @@
 
 
 class MessageDispatcher(object):
-    def __init__(self, slackclient, plugins):
+    def __init__(self, slackclient, plugins, default_reply=None):
         self._client = slackclient
         self._pool = WorkerPool(self.dispatch_msg)
```

Comment:
It's fine for each bot to have a worker pool, but we may improve it later so that all the instances share the same pool to reduce the total threads created (currenlty `WorkerPool` would create 10 threads at construction time).